### ISSUE TITLE
After login, once a few maps are selected to make a project, “maps” icon on left

### ISSUE THREAD

**Issue by AshleyCheyemi** [emory-libraries-ecds/ATLMaps-Ember#3]
-- “explore” on top of extended box → what does that mean?
-- Small icon of boxes in each map selection of extended box for “maps” → are these icons supposed to do something?
-- What is the sliding on/off-looking button on top right of extended box do?

**Comment by jayvarner** [comment]
@AshleyCheyemi do you mind adding a screenshot?

**Comment by jayvarner** [comment]
- _“explore” on top of extended box → what does that mean?_

That space is normally where the project title is. In this case, the project it titled _Explore_. Clearly that wasn't, well, clear. Maybe we can replace with `Log in or register to save`?

- _Small icon of boxes in each map selection of extended box for “maps” → are these icons supposed to do something?_

Not sure what you mean. Can you add a screenshot?

- _What is the sliding on/off-looking button on top right of extended box do?_

It _should_ hides the points/lines/polygons of that layer. This is fixed in the next code push.

**Comment by AshleyCheyemi** [comment]
Small icon of boxes in each map selection of extended box for “maps” → are these icons supposed to do something? Here's what I mean:
![screen shot 2017-07-07 at 4 50 24 pm](https://user-images.githubusercontent.com/29981780/27976631-62f93b6e-6335-11e7-888f-867d29187d1e.png)

**Comment by jayvarner** [comment]
Yeah, this is tricky. This is where you can grab a layer and reorder the maps. As a civilization, we have not come to consensus on what icon says "drag me." ![https://github.com/FortAwesome/Font-Awesome/issues/816](https://github.com/FortAwesome/Font-Awesome/issues/816)

Or just google things like icon drag or sort or reorder.

![Gif of reordering layers.](https://media.giphy.com/media/l4FGulPqbhS088cve/giphy.gif)

**Comment by jayvarner** [comment]
@AshleyCheyemi let me know your thoughts on this.

**Comment by AshleyCheyemi** [comment]
I see what you mean. I kind of prefer opposing arrows, like this link below, but we can ask the group: https://oit.colorado.edu/sites/default/files/images/reorder.png
